draft-bbs-signatures.md

```diff
@@ -181,6 +181,9 @@ I \\ J
 X\[a..b\]
 : Denotes a slice of the array `X` containing all elements from and including the value at index `a` until and including the value at index `b`. Note when this syntax is applied to an octet string, each element in the array `X` is assumed to be a single byte.
 
+(a, ..., b)
```

Comment:
```suggestion
(a..b)
```

Reply:
OR something like python `range(0,2)`

Reply:
Thanks! Updated to `range(0, 2)` (we use [a..b] for array/octet_string slices)